public/dorequest.php

```diff
@@ -219,11 +221,6 @@ function DoRequestError(string $error, ?int $status = 200, ?string $code = null)
     /*
      * Global, no permissions required
      */
-    case "allprogress":
```

Comment:
Technically, this does require authentication. `$user` only gets set by processing the token. I've made that more clear with this refactor by inheriting from `BaseAuthenticatedApiAction`.

And I can't think of any we'd want to query all progress for a console for some other user (particuarly through the Connect API).